framework/include/meshgenerators/DistributedRectilinearMeshGenerator.h

```diff
@@ -303,6 +303,9 @@ class DistributedRectilinearMeshGenerator : public MeshGenerator
   /// The min/max values for x,y,z component
   Real &_xmin, &_xmax, &_ymin, &_ymax, &_zmin, &_zmax;
 
+  /// Number of cores for partitioning the graph
+  dof_id_type _num_cores_for_partition;
```

Comment:
Is this the right type or should it be `processor_id_type`?

Reply:
Used processor_id_type